src/Core/Registration/Registration.cpp

```diff
@@ -146,8 +146,16 @@ RegistrationResult RegistrationICP(const PointCloud &source,
         const ICPConvergenceCriteria &criteria/* = ICPConvergenceCriteria()*/)
 {
     if (max_correspondence_distance <= 0.0) {
+        PrintError("Error: Invalid max_correspondence_distance.\n");
```

Comment:
Do we have some standard form of error message? In this example, it is just ``error: [message]``

Reply:
I usually use "[function_name] some_message".